### ISSUE TITLE
Disallow floats as locations? Round?

### ISSUE THREAD

**Issue by madig** [daltonmaag/statmake#16]
Due to how floats work, using them as locations is tricky. A Designspace location of `569.078000` and a Stylespace location of `569.078` may result in an error like "There is no Stylespace entry for stop 569.0780029296875 on the 'wdth' axis.".

**Comment by anthrotype** [comment]
No, locations must be floats. In VF they are encoded as Fixed type. Maybe rounding to 3 or 4 digits should be enough.

**Comment by weiweihuanghuang** [comment]
I had this issue
